docassemble/HousingCodeChecklist/data/questions/customized_screens.yml

```
@@ -121,21 +127,42 @@ if: |
   landlord_name_is_optional
 id: names of opposing parties name not required
 question: |
-  What is your landlord's name?
+  % if person_answering == "tenant":
+  Please answer some questions about your landlord
+  % else:
+  Please answer some questions about the tenant's landlord
+  % endif
 subquestion: |
+  % if person_answering == "tenant":
+  First, you will be asked if your landlord is a business or an individual and then what your landlord's name is.
+  
   Write the name of your landlord, not your property manager. The landlord
   is the person who owns the building. Usually it is also the name you
   write on a rent check.
+  % else:
+  First, you will be asked if the tenant's landlord is a business or an individual and then what the landlord's name is.
+  
+  Write the name of the tenant's landlord, not the property manager. The landlord
+  is the person who owns the building. Usually it is also the name the tenant
+  writes on a rent check.
+  % endif
 fields:
-  - I do not know my landlord's name: ll_name_unknown
+  - ${"I do not know my landlord's name" if person_answering == "tenant" else "I do not know the name of the tenant's landlord"}: ll_name_unknown
     datatype: yesno
   - note: |
+      % if person_answering == "tenant":
       **Okay**. You can still fill in this form, but you will need
       to write in your landlord's name on every form before you deliver
       it. You may be able to find their name by visiting the Registry of
       Deeds for your city or town.
+      % else:
+      **Okay**. You can still fill in this form, but the tenant will need
+      to write in the name of their landlord on every form before they deliver
+      it. You may be able to find the name of the tenant's landlord by visiting the Registry
+      of Deeds for the tenant's city or town.
+      % endif
     show if: ll_name_unknown
-  - Is your landlord an individual or a business?: other_parties[0].person_type
+  - ${"Is your landlord an individual or a business?" if person_answering == "tenant" else "Is the tenant's landlord an individual or a business?"}: other_parties[0].person_type
```

Slight preference to write this out longhand vs using a 1-line if statement/ternary (it's a bit easier to get the translator to handle correctly):
```suggestion
  - label: |
      % if person_answering =="tenant":
      Is your landlord an individual or a business?
      % else:
      Is the tenant's landlord an individual or a business?
      %endif
    field: other_parties[0].person_type
```